build_android/res/values/strings.xml

```diff
@@ -1,35 +1,71 @@
 <resources>
-	<string name="app_name">Play!</string>
+    
+    <string name="app_name">Play!</string>
+    
+    <!-- Application Navigation -->
+    
 	<string name="main_menu_settings">Settings</string>
 	<string name="main_menu_about">About</string>
 	<string name="main_menu_exit">Exit</string>
 
 	<string name="game_info">No info</string>
+    
+    <string name="search_games">Locating games&#8230;</string>
+    <string name="search_games_msg">Please wait&#8230;</string>
+    
+    <string name="drawer_open">Menu Open</string>
+    <string name="drawer_shut">Menu Shut</string>
+    
+    <string name="menu_title_open">Menu</string>
+    <string name="menu_title_shut">Games</string>
+    <string name="menu_title_look">Search</string>
+    
+    <string name="file_list_recent">Recent</string>
+    <string name="file_list_homebrew">Homebrew</string>
+    <string name="file_list_default">Unsorted</string>
+    
+    <!-- Applciation Preferences -->
+    
+    <string name="title_activity_settings">Settings</string>
+    
+    <string name="preferences_screen_emu">Emulator Settings</string>
+    
+    <string name="pref_emu_fps_display_title">Display F/S &amp; DC/F</string>
+    <string name="pref_emu_fps_display_summary">Show \"Frames Per Second\" and \"Draw Calls Per Frame\" on screen</string>
+    
+    <string name="pref_emu_pad_display_title">Show Virtual Pad</string>
+    <string name="pref_emu_pad_display_summary">Show virtual pad on screen</string>
+    
+    <string name="preferences_screen_ui">UI Configuration</string>
+    
+    <string name="preferences_ui">Storage Options</string>
+    
+    <string name="pref_ui_reset_folder_title">Reset Folder</string>
+    <string name="pref_ui_reset_folder_summary">Clear stored default folder</string>
+    
+    <string name="pref_ui_reset_cache_title">Reset Cache</string>
+    <string name="pref_ui_reset_cache_summary">Clear stored cover images</string>
+    
+    <string name="preferences_theme">Theme Settings</string>
+    
+    <string name="pref_title_theme_selection">Theme Selection</string>
+
+    <string-array name="pref_theme_selection_titles">
+        <item>Amber</item>
+        <item>Blue (Default)</item>
+        <item>Pink</item>
+        <item>Purple</item>
+        <item>Teal</item>
+    </string-array>
+    
+    <!-- Error Reporting / Debugging -->
 	
-	<string-array name="disks">
-		<item>iso</item>
-		<item>bin</item>
-		<item>cso</item>
-		<item>isz</item>
-		<item>elf</item>
-	</string-array>
-	
-	<string-array name="homebrew">
-		<item>elf</item>
-	</string-array>
-	
-	<string name="search_games">Locating games...</string>
-	<string name="search_games_msg">Please wait...</string>
+	<string name="report_issue">Previous Crash Detected</string>
 	
-	<string name="menu_title_open">Menu</string>
-	<string name="menu_title_shut">Games</string>
-	<string name="menu_title_look">Search</string>
-
-    <string name="navigation_drawer_open">Navigation drawer open</string>
```

Comment:
Why was this line removed? It still seems to be used here: https://github.com/Thunder07/Play-/blob/theme/Source/ui_android/java/com/virtualapplications/play/NavigationDrawerFragment.java#L168